data/ecosystems/s/sns.toml

```diff
@@ -0,0 +1,24 @@
+# Ecosystem Level Information
+title = "Sui Name Service (SNS)"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/snsdomains"]
+
+# Repositories
+[[repo]]
+url = "https://github.com/snsdomains/js"
+
+[[repo]]
+url = "https://github.com/snsdomains/objects.json"
+
+[[repo]]
+url = "https://github.com/snsdomains/objects.json"
+
+# Private Repositories (will be made public)
+
+[[repo]]
+url = "https://github.com/snsdomains/domains"
+
+[[repo]]
+url = "https://github.com/snsdomains/web"
```

Comment:
Let's remove these for now since we only track open source repos in the taxonomy at the moment.  The github_organization being specified above will automatically add them in the future when they are open sourced.

Reply:
Sounds good. I just removed them. Please check now